text/0000-edition-2021-and-beyond.md

```diff
@@ -76,9 +79,11 @@ Along with each edition, we also release support for it in a tool called `rustfi
 
 When we release a new edition, it comes together with a certain set of "migrations". Migrations are the "breaking changes" introduced by the edition, except of course that since editions are opt-in, no code actually breaks. For example, if we introduce a new keyword, you will have to rename variables or functions using the old keyword, or else use Rust's `r#keyword` feature (which allows you to use a keyword as a regular variable/function/type name). As mentioned before, the edition comes with tooling that will make these changes for you, though sometimes you will want to cleanup the resulting code afterwards.
 
-Editions can also change the default severity of lints, so that instead of defaulting to "warn", they default to "deny" for code using the new edition. This is done to help encourage deprecation of language features or patterns that have been found to be harmful. Because lints will now default to "deny", they can feel like other migrations, but there is an important difference -- you can opt to change the lint level back to "warn" or "allow" if you don't want to change the code yet.
+Migrations target the "intersection" of editions. That is, a migration will convert code in Edition N into code that compiles in **both** Edition N and Edition N+1. This permits migrations to be applied in a piecemeal fashion rather than forcing all migrations to occur at once. 
 
-Lints whose severity level changes with an edition are called "idiom lints". For idiom lints associated with Edition N will be warn-by-default for earlier editions, but become deny-by-default for code that opts into Edition N. (Note that the lints are typically introduced long before the edition itself, and they simply issue warnings until the Edition is released.)
+Because migrations produce code that compiles in both Edition N and Edition N+1, that code may not be idiomatic in the new edition. To address this, each edition can also include **idiom lints**. These are lints which identify deprecated or altered patterns from older editions and convert them into the newer style. Some of these lints will produce code that that is only valid in edition N+1, in which case the lints will only be triggered in the new edition.
```

Comment:
Small typo:

```suggestion
Because migrations produce code that compiles in both Edition N and Edition N+1, that code may not be idiomatic in the new edition. To address this, each edition can also include **idiom lints**. These are lints which identify deprecated or altered patterns from older editions and convert them into the newer style. Some of these lints will produce code that is only valid in edition N+1, in which case the lints will only be triggered in the new edition.
```